### PR TITLE
Oskar gdev 2548 risk scoring tool

### PR DESCRIPTION
**Pull request by oskar700**
# Pull request questions

## Which issue does this address 

Issue number: GDEV-2548

## Why was change needed

This is to service customer need.

## What does change improve

Adds ability to monitor the quality of data.

### CODE REVIEW

File: src/main/java/com/senzing/calculator/scoring/risk/service/RiskScoringService.java [hunk 1]
```diff
@@ -0,0 +1,370 @@
+package com.senzing.calculator.scoring.risk.service;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
```
Comment (barrycaceres) [inline src/main/java/com/senzing/calculator/scoring/risk/service/RiskScoringService.java:11]: as an FYI, Java does have a standard JSON parsing api (javax.json).  Don't know if using org.json is a requirement or not -- for the API server we use org.glassfish implementation of javax.json.

File: .project [hunk 2]
```diff
@@ -1,4 +1,23 @@
...
+			<name>org.eclipse.m2e.core.maven2Builder</name>
+			<arguments>
+			</arguments>
+		</buildCommand>
+	</buildSpec>
+	<natures>
+		<nature>org.eclipse.jdt.core.javanature</nature>
+		<nature>org.eclipse.m2e.core.maven2Nature</nature>
+	</natures>
 </projectDescription>
```
[the remark this answers is not in the document]
Comment (oskar700) [reply .project:23]: I added .project into .gitignore.   Also backed the .project change out.

File: pom.xml [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+  <dependencies>
+    <dependency>
+      <groupId>com.senzing</groupId>
+      <artifactId>senzing-listener</artifactId>
+      <version>0.0.1-SNAPSHOT</version>
+    </dependency>
+    <dependency>
+      <groupId>org.json</groupId>
+      <artifactId>json</artifactId>
+      <version>20090211</version>
```
[the remark this answers is not in the document]
Comment (oskar700) [reply pom.xml:22]: I replaced the parser and use not the one you recommend.

File: src/main/java/com/senzing/calculator/scoring/risk/service/RiskScoringService.java (hunk 1 above)
Comment (oskar700) [reply src/main/java/com/senzing/calculator/scoring/risk/service/RiskScoringService.java:11]: I replaced the parser.

**Comment by oskar700** [comment]
Thanks for reviewing Barry.  Regarding the destroy() call, there is only one instance that lives for the lifetime of the app, but I added this and call on exit.  It will be good if more threads, or functionality changes in the future.
I replaced the json.org parser with java.json.

**Pull request merged**